composer.json

```diff
@@ -15,8 +15,9 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/security": "~2.3|~3.0",
+        "php": ">=5.5.9",
+        "symfony/framework-bundle": "~3.0",
+        "symfony/security": "~3.0",
```

Comment:
you should depend on actual security subcomponents instead of using the bigger component